src/main/scala/io/github/sentenza/hacktoberfest18/sort/CommonSortings.scala

```diff
@@ -17,6 +17,7 @@ package io.github.sentenza.hacktoberfest18.sort
 
 object CommonSortings {
   trait CommonSorting {
+    def swap(array: Array[Int], index1: Int, index2: Int)
```

Comment:
Wouldn't it be better to define the swap() function only in the companion object? It should be just an helper method, right?

Reply:
yes:)